src/_pytest/reports.py

```diff
@@ -304,7 +309,7 @@ def __init__(self, nodeid, outcome, longrepr, result, sections=(), **extra):
         self.nodeid = nodeid
         self.outcome = outcome
         self.longrepr = longrepr
-        self.result = result or []
+        self.result = result or []  # type: List
```

Comment:
Do you what type of list it is?

Reply:
Via CallInfo.from_call usually: https://github.com/blueyed/pytest/blob/1b3c0e6d5c078dadd11dfb2a109dc330742ac75c/src/_pytest/runner.py#L210-L236

Reply:
So can this be `List[CallInfo]`?

(btw, it's perfectly fine if you left it out because it's too much work to check, I'm just commenting in case you forgot to fill it in).